packages/components/src/tools-panel/tools-panel-header/component.tsx

```diff
@@ -22,9 +18,14 @@ import MenuGroup from '../../menu-group';
 import MenuItem from '../../menu-item';
 import { useToolsPanelHeader } from './hook';
 import { contextConnect, WordPressComponentProps } from '../../ui/context';
-import type { ToolsPanelHeaderProps } from '../types';
+import type {
+	ToolsPanelControlsGroupProps,
+	ToolsPanelHeaderProps,
+} from '../types';
+
+const DefaultControlsGroup = ( props: ToolsPanelControlsGroupProps ) => {
```

Comment:
It shouldn't be necessary to split out the destructuring of the props in this case (when they're being actually typed), that was just a strategy to prevent TS from inferring specific types for the props of the components we didn't want it to check.

Reply:
This has been addressed, thanks!